### DISCUSSION TITLE
Zusätzliche Informationen auf Rückseite des Titelblatts

### DISCUSSION THREAD

**Discussion by GGottschalg** [tudace/tuda_latex_templates#387] [Q&A] [answered]
Hallo, 

für meine Dissertation würde ich gerne auf der Rückseite des Titelblatts weitere Veröffentlichungsorte nennen. Gibt es ein Feld mit dem ein Text nach dem Teil zu "Dieses Dokument wird bereitgestellt von tuprints, E-Publishing-Service der TU Darmstadt http://tuprints.ulb.tu-darmstadt.de tuprints@ulb.tu-darmstadt.de" und vor der Angabe der Lizenz eingefügt werden kann?

Vielen Dank im Vorraus!

Viele Grüße
Grischa

**Comment by TeXhackse** [comment, marked as answer]
Oh, Irgendwie ist mir das Durchgerutscht, ich bin noch nicht an die Discussions gewöhnt. 

Grundsätzlch ist die Seite schon da. Wichtig ist, dass das Titelblatt natürlich nur eine Rückseite hat, wenn das Dokument doppelseitg ist.

**Reply by TeXhackse** [reply]
Hat das soweit geholfen oder ist da noch etwas unklar?

**Reply by GGottschalg** [reply]
Ja, ich habe das Problem lösen können. Wir können diesen Thread schließen. Danke.
